nameko/testing/services.py

```diff
@@ -286,30 +299,32 @@ def restrict_entrypoints(container, *entrypoints):
 
     **Usage**
 
-    The following service definition has two entrypoints for "method"::
+    The following service definition has two entrypoints:
+
+    .. code-block:: python
 
         class Service(object):
 
-            @rpc
+            @timer(interval=1)
             def foo(self, arg):
                 pass
 
             @rpc
-            @event_handler('srcservice', 'event_one')
             def bar(self, arg)
                 pass
 
         container = container_factory(Service, config)
 
-    To disable the entrypoints other than on "foo"::
-
-        restrict_entrypoints(container, "foo")
+    To disable the timer entrypoint on ``foo``, leaving just the RPC
+    entrypoint on ``bar``:
 
-    To maintain both the rpc and the event_handler entrypoints on "bar"::
+    .. code-block:: python
 
         restrict_entrypoints(container, "bar")
```

Comment:
You can also provide a list of entrypoint names, should you want to keep several, is this right? Might be worth mentioning/showing an example in this docstring.